src/orders/components/OrderDraftDetailsProducts/TableLineForm.tsx

```diff
@@ -35,8 +30,15 @@ const TableLineForm: React.FC<TableLineFormProps> = ({
   const classes = useStyles({});
   const { id, quantity } = line;
 
+  const handleSubmit = (id: string, data: OrderLineInput) => {
+    if (!data || data.quantity < 1) {
+      data.quantity = 1;
```

Comment:
you're mutating state here, this can lead to unexpected React bugs. If you want to change the value, you should use `change` setter instead:
```
  const handleSubmit = (id: string, data: OrderLineInput) => {
    const quantity = data?.quantity >= 1 ? Math.floor(data.quantity) : 1;

    onOrderLineChange(id, { quantity });
```

Reply:
Ah yes, forgot here that data object is a state. Thanks